src/test/mochitest/browser_dbg-returnvalues.js

```diff
@@ -0,0 +1,61 @@
+function getLabel(dbg, index) {
+  return findElement(dbg, "scopeNode", index).innerText;
+}
+
+function getValue(dbg, index) {
+  return findElement(dbg, "scopeValue", index).innerText;
+}
+
+function toggleScopes(dbg) {
+  return findElement(dbg, "scopesHeader").click();
+}
+
+async function testReturnValue(dbg, val) {
+  invokeInTab(dbg, "return_something",val);
```

Comment:
We're sending 3 args here, while `invokeInTab` only accepts two; don't think we need `dbg`.

Reply:
...actually, simply removing that doesn't fix test failures.  Still looking into it.